kfdefs/overlays/moc/zero/opf-monitoring/datasource/opf-openshift-monitoring-grafanadatasource.enc.yaml

```diff
@@ -0,0 +1,53 @@
+apiVersion: integreatly.org/v1alpha1
+kind: GrafanaDataSource
+metadata:
+    name: opf-openshift-monitoring-grafanadatasource
+    namespace: opf-monitoring
+    labels:
+        app: grafana-datasources
+        app.kubernetes.io/instance: kfdefs-zero
+        component: operate-first
+spec:
+    datasources:
+    -   editable: false
+        jsonData:
+            httpHeaderName1: Authorization
+            timeInterval: 5s
+            tlsSkipVerify: true
+        name: openshift-monitoring
+        secureJsonData:
+            httpHeaderValue1: ENC[AES256_GCM,data:ZSCGAiejcdT2BDch+yESQdEgDp5Qixp0H3S/iS1PwBX3tIdT1dszkdlycjzvdZRO762qHcEXEN9gmo/jiwoKD3XgWHDP2zlsCzZdMhJ3Vu7s8D/XvcoHdEoTD/pVs3S3Biu8ib9D1ePS2g/ttWRPlFv7zaRbj3KR07OS0xtNFr9x0d+aImoTPZtqmx8ppskEchHU/Jk4KtsK+khdfe1zlCdov8ADuMz75RSQBjRBhGzNwns8TdjU3fJDUB9eYAc/RJhGr4UA2YE3TQtiq7YQno8SDfGUvGfHd03mEpGueeulO+rMVYzJmkoC61e3v1/Xkqr55D9iB8XRSA9sFEdk2cml0NWGR/Du9JJ2UuzJ5Rg9IDuL6YYtq3AGzcB51hnfcYPPb9VHApZnEdYm/Mv8+7SEFK7Gm8ti1ESyntjOgZYdUwXqG9UrOrJmSEDLDGMp2JCT4QB87uKzQIjecg3i2MQYUXeZljgrmNNHi3JRT5l5r4z8MPASd24zX7NxoC8woijpVXmfgyeH+kTRc13oramzvw2or23NMK4ejnBRH45/NjsDGQ2QJRpr8o8r/UQ7lKifTE90keHRtF0LkTHugLBgNDloNz7UAtAUIA29tKg/9V0RP4xIw3a7SZY4xSMKBl/LtUia1CPaRbZLDky41LMtP2U3iYU6U3XWx3QkNOZkt6+2/qfScl+mERo1Sde7uzkrNyQXrwBcoVlBL/LSmnAw9LIW57rmHy2AW6EKHpZAUuwa7vwDX3GcgnCMNA1DIiiP5pZgM0rk8y89lrLdY3HvU91mIojr/mlJxtMsg59N6unapQ7YxoHBu+9zD7gS6nQLdoCwaFSn5K5n128UotNHJA+kRSsLSZMZsSu+BWhwea4vkA5ZOSPzc+A+tR79azGwvjx8i1xW9YVCaRYyqSbR6/d2yMZMj2TE9FNzkcygvzFQLXq4hl3IAdmK4m1ywWj5IelcnOhZ51vTiVxDtcvwSCx+OwNRbIPhKpzddatgUPn8/4KykhCJ+JLv7ljmcqMsqfTe4QZfcbX52PJ/UOH0w5V+Xe0qGQ5tbklLdAcT3BiF7fc8lJUKoPDwSeqPStQFnfJV73gR/rKKG1VF4En5byzL0sRMDPjn/IlcCjCOMHHroI1LP2HeA2bKq9bf5ROUjmki7PjxC1VGU9S6oQly+4OC4sOOw00Mqsi30R/OVrx2fIJMzwEa+q8sccXj6u8E5TyIbHqQ0BTXrHk2jueV1io3FfTi8UHNtUbsPN0IJy04SsgOBO5GNnHOMzIcjnfzvn/lAhPMtRM7IG5GzWKF7m9FOEXv9g==,iv:NCgRuUiYecJqAfVRorr+wzW5bjTUz1san+KxjgyPAIQ=,tag:cDopAmZKnGeJLtTSO5Q13A==,type:str]
+        type: prometheus
+        url: https://thanos-querier.openshift-monitoring.svc.cluster.local:9091
```

Comment:
do we want to connect to the openshift-monitoring Thanos instance and not the Prometheus instance?

Reply:
I'm not sure, the connected issue stated `openshift-monitoring prometheus` 
@tumido ?

Reply:
The blog inked in the issue was forwarded to me as "the proper" solution. It suggests to connect to thanos querier. I think this is correct solution... Let's try it this way. :slightly_smiling_face:

Reply:
We should also track https://github.com/integr8ly/grafana-operator/issues/189 to make the token loading dynamic from secret.

Reply:
good call, I've subscribed to that thread